website/src/components/stackblitz.js

```diff
@@ -1,6 +1,6 @@
 export const Stackblitz = ({ id, file }) => {
   return (
-    <div className="mb-8 overflow-hidden rounded-md border-b border-gray-200 shadow-lg dark:!shadow-none sm:rounded-lg">
+    <div className="mb-8 mt-4 overflow-hidden rounded-md border-b border-gray-200 shadow-lg dark:!shadow-none sm:rounded-lg">
```

Comment:
FYI, this would add some top margin to the Stackblitz block to keep it from too close to the heading above.
<img width="962" alt="Screenshot 2024-04-26 at 4 52 21 PM" src="https://github.com/pmndrs/jotai/assets/234226/775c2b67-6622-424b-a76f-7eca038454b3">